collection-scripts/version

```diff
@@ -1,18 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Is it intentional to keep this file here just empty?

Reply:
must be mistakenly checkin after function has been moved away